akka-typed/src/main/scala/akka/typed/patterns/Receptionist.scala

```diff
@@ -72,18 +72,20 @@ object Receptionist {
 
   private type KV[K <: AbstractServiceKey] = ActorRef[K#Type]
 
-  private def behavior(map: TypedMultiMap[AbstractServiceKey, KV]): Behavior[Command] = Full {
-    case Msg(ctx, r: Register[t]) ⇒
+  private def behavior(map: TypedMultiMap[AbstractServiceKey, KV]): Behavior[Command] = Stateful[Command]({
+    case (ctx, r: Register[t]) ⇒
```

Comment:
Didn't think it mattered here. Doesn't the tuple unhappy also cause an allocation?

When I look at the byte code from `class B(t: (String, Int)) { val (a, b) = t }` I see:
```
    37: invokespecial #53                 // Method scala/Tuple2."<init>":(Ljava/lang/Object;Ljava/lang/Object;)V
```